src/KeyVault/KeyVault.Test/ScenarioTests/KeyVaultManagementTests.cs

```diff
@@ -95,14 +60,7 @@ public void TestPublicNetworkAccessWhenCreateNewVault()
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestGetVault()
         {
-            KeyVaultManagementController.NewInstance.RunPsTestWorkflow(
-                _logger,
-                () => { return new[] { "Test-GetVault" }; },
-                null,
-                MethodBase.GetCurrentMethod().ReflectedType?.ToString(),
-                MethodBase.GetCurrentMethod().Name,
-                Initialize);
-
+            TestRunner.RunTestScript("Test-GetVault");
```

Comment:
This test case can successfully pass after removing "Initialize" function. Does it still needs to be skipped?